xtc/src/history.rs

```diff
@@ -71,12 +75,72 @@ impl HistoryBuffer {
     pub fn len(&self) -> u64 {
         self.history.len()
     }
+
+    pub fn get_history(&self) -> &History {
```

Comment:
The standard for rust "getters" is to just use the member name, in every project I've been on and this idea is shared between all devs I've asked.

```suggestion
    pub fn history(&self) -> &History {
```